layouts/partials/content.html

```diff
@@ -24,9 +24,27 @@
 
       <h2 id="available-scalers">
         Currently available scalers for KEDA
-      </h2>
-      {{ partial "scalers.html" . }}
+      </h2>      
+      <div class="buttons" style="display: flex; justify-content: flex-end">
+        <form class="control has-icons-left has-extra-sm-right-margin" id="search" role="search">
+          <input class="input is-rounded" type="search" placeholder="Search for scalers" id="search-input">
+          <span class="icon is-small is-left">
+            <i class="fas fa-search"></i>
+          </span>
+        </form>        
+        <button id="btn-focus-on-mount" class="button filterBtn is-focused" value="built-in-scalers">Built-in</button>
+        <button class="button filterBtn" value="external-scalers">External</button>
+      </div>
+      <div class="results"></div>
+      <div id="built-in-scalers" style="margin-top:20px;" class="columns is-multiline is-search-result">
+        <template id="is-search-template">
+        {{ .Render "scaler-template" }}
+        </template>
+      </div>
+      <div id="external-scalers" class="artifacthub-widget-group" style="display: none;" data-url="https://artifacthub.io/packages/search?kind=8&sort=relevance&page=1" data-theme="light" data-header="false" data-stars="true" data-color="#405273" data-responsive="true"  data-loading="true"></div>
```

Comment:
We should add a section here that explains that external scaler information is pulled from Artifact Hub.

External scaler maintainers can learn more on https://artifacthub.io/docs/topics/repositories/#keda-scalers-repositories

Reply:
Reactivating this as this is just a comment but the idea was to show this to our end-users

Reply:
Do you mean something like a notice? @tomkerkhove

Reply:
Just a sentence would be enough; just like the introduction.

Reply:
If so, I will suggest we create this as a separate issue as this PR is growing out of its original scope. @tomkerkhove

Reply:
> Just a sentence would be enough; just like the introduction.
Oh...I see. Alright then, will push something